feat: add orm client to typescript webhooks

leoalves opened the pull request:
<!--
Thank you for your pull request. Please provide a description above and review
the requirements below.

Bug fixes and new features should include tests and possibly benchmarks.

Contributors guide: https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md
Squashed commit must follow https://www.conventionalcommits.org/en/v1.0.0/ so we can generate the changelog.

By making a contribution to this project, I certify that:

* (a) The contribution was created in whole or in part by me and I
  have the right to submit it under the open source license
  indicated in the file; or

* (b) The contribution is based upon previous work that, to the best
  of my knowledge, is covered under an appropriate open source
  license and I have the right under that license to submit that
  work with modifications, whether created in whole or in part
  by me, under the same open source license (unless I am
  permitted to submit under a different license), as indicated
  in the file; or

* (c) The contribution was provided directly to me by some other
  person who certified (a), (b) or (c) and I have not modified
  it.

* (d) I understand and agree that this project and the contribution
  are public and that a record of the contribution (including all
  personal information I submit with it, including my sign-off) is
  maintained indefinitely and may be redistributed consistent with
  this project or the open source license(s) involved.
-->

## Motivation and Context

<!--
Why is this change required? What problem does it solve? Which issues are linked?
Please try to describe in detail the impact of this change. Add screenshots if it helps.
-->
Currently Typescript Webhooks do not expose the new ORM.  This pulls request adds the ORM to the Webhook context

## Marketing Changelog

<!--
What changes does this PR introduce? Please describe the changes in simple terms that a user can understand
without being familiar with the codebase. Mention @advocates for review and tracking.
-->

#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

jensneuse reviewed (approved):
I think it looks good overall. We might want to add a test to the teatsuite package that uses the ORM within a webhook to fixate this feature.

leoalves commented:
> I think it looks good overall. We might want to add a test to the teatsuite package that uses the ORM within a webhook to fixate this feature.

I can try to do that. Just need to figure it out how the test suite works.  I probably won't have time this weekend but should be able to do it Monday.
Thanks

leoalves commented:
I added a new project to the testsuite package (`/packages/testsuite/apps`) to test the addition of the ORM to the webhook.  But when the webhook calls the GraphQL server it returns 404. My guess is that the test server doesn't have the GraphQL server.  The test just throws an error that it's not able to form the JSON. This might be a bug in the ORM package, it's not validating if the response has the `.data` key or not.  

Is there another way to test it the ORM in the webhook?  I saw that there is another folder (`/testapps/default`).  In that project webhook I added a call to the ORM.  But I am not sure how to test it.  I didn't see anywhere where those operations are called from a test file.   

Thanks

jensneuse commented:
@leoalves thank you for contributing this! We'll make a release soon.

leoalves commented on pkg/apihandler/internalapihandler.go:
```diff
@@ -124,38 +121,18 @@ func (i *InternalBuilder) BuildAndMountInternalApiHandler(ctx context.Context, r
 		}
 	}
 
-	if api.EnableGraphqlEndpoint {
```
Does this mean the `/graphql` endpoint will be public now?

jensneuse replied:
No, it's internal.
